dotCMS/src/main/java/com/dotcms/vanityurl/model/DefaultVanityUrl.java

```diff
@@ -18,10 +18,6 @@
      */
     private static final long serialVersionUID = 1L;
 
-    @Override
-    public String getTitle() {
```

Comment:
why did you remove this method?

Reply:
we redesign the logic; not the getTitle is final. No body can override it. This is to avoid weird behaviour such as the contentlets modified here, that the title generating didn't follow the dotCMS logic